### ISSUE TITLE
Move GitHub actions to Alice

### ISSUE THREAD

**Issue by krisnova** [aurae-runtime/aurae#376]
Right now we are unable to load/test our eBPF probes because of the limitation with GitHub actions. 

Can we move to our own GitHub actions runners so we can load/test eBPF and virtualization in the future?

**Comment by jesse-peters** [comment]
Should we consider running them using kubernetes [actions-runner-controller](https://github.com/actions/actions-runner-controller), or do we need to run on an actual host for these tests?

**Comment by JeroenSoeters** [comment]
Do we have any infra we could leverage for this? I wouldn't mind spending some time on this. I think we should run those tests on every PR to ensure we don't introduce any regressions.

**Comment by krisnova** [comment]
Yeah we have a rack of large bare metal servers we can give you access to -- message me on discord and I can get you access and in touch with the folks who manage it

**Comment by dmah42** [comment]
i believe Nova was referencing the servers we use to host hachyderm.io.  i'm not sure we would want to use the same servers, but spinning up a server to run this is certainly doable.

**Comment by dmah42** [comment]
we run the cla-bot on a digital ocean box using kubectl.  wouldn't it be nice to run it, and github action runners, using auraed...
